pan_kinect:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: pan_kinect
    joint_speed: 0.4
    motor:
        id: 10
        init: 2380
        min: 0
        max: 4095

tilt_kinect_2:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_kinect_2
    joint_speed: 0.4
    motor:
        id: 11
        init: 2240
        min: 1500
        max: 2220

tilt_kinect:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_kinect
    joint_speed: 0.4
    motor:
        id: 11
        init: 2000
        min: 1575
        max: 2035

tilt_laser:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: tilt_laser
    joint_speed: 1.0
    motor:
        id: 1
        init: 214
        min: 20
        max: 856
        
mark44_1:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: mark44_1
    joint_speed: 0.4
    motor:
        id: 21
        init: 23537
        min: 1000
        max: 64500

mark44_2:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: mark44_2
    joint_speed: 0.4
    motor:
        id: 22
        init: 24411
        min: 1000
        max: 64500

mark44_3:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: mark44_3
    joint_speed: 0.4
    motor:
        id: 23
        init: 66500
        min: 26130
        max: 55726

mark43_pris:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: mark43_pris
    joint_speed: 0.4
    motor:
        id: 26
        init: 0
        min: 0
        max: 8192

joint1:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint1
    joint_speed: 0.4
    motor:
        id: 40
        init: 2048
        min: 0
        max: 4095          
        
joint3:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint3
    joint_speed: 0.4
    motor:
        id: 42
        init: 2048
        min: 0
        max: 4095
        
joint2:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: joint2
    joint_speed: 0.4
    motor:
        id: 41
        init: 403
        min: 65
        max: 420
                
gripper:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: gripper
    joint_speed: 0.4
    joint_torque_limit : 0.25
    motor:
        id: 43
        init: 2048
        min: 1786
        max: 2587
